src/messages.js

```diff
@@ -95,7 +95,7 @@ var warnings = {
   W018: "Confusing use of '{a}'.",
   W019: "Use the isNaN function to compare with NaN.",
   W020: "Read only.",
-  W021: "'{a}' is a function.",
+  W021: "'{a}' is a {b}.", // is a class or is a function
```

Comment:
Thanks, it is present in the regressions of two libraries, but yes no unit tests.

I've changed the message (your suggestion is alot better, I noticed it wasn't very informative but wasn't sure it was a good idea to change it). I've done it in a seperate [[CHORE]] commit - upto you if you want to squash it or not.